### DISCUSSION TITLE
Consistency in wording

### DISCUSSION THREAD

**Discussion by TheDauntless** [OWASP/masvs#585] [Ideas]
The controls are worded in a few different ways right now.

Most of them are affirmative:

> No sensitive data is written to application logs.

But some are conditional:

> The app’s local storage should be wiped after an excessive number of failed authentication attempts.

I propose to rewrite the second type to the first type:

> The app’s local storage is wiped after an excessive number of failed authentication attempts.

I can pick this up if you guys agree :)

**Comment by commjoen** [comment]
I rather wait with new requirements till some of the translational work is done :). Let's start updating master with new requirements around July 👍

**Comment by commjoen** [comment]
Getting this right and consistent cross-cultural in our different translations will also be quiet a large challenge. So yes, I agree that the wording is different, but I wonder if we want to pay the price in all the translations to get the cultural differences in the language across.
